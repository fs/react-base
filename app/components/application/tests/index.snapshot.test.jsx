import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { fakeSession } from 'mocks/fakeSession';
import Application from '../';

describe('Application', () => {
  let props;
  const renderComponent = () => shallow((
    <Application {...props}>
      <div />
    </Application>
  ));

  beforeEach(() => {
    props = {
      logout: () => {},
      session: fakeSession,
      signin: () => {},
      signup: () => {},
    };
  });

  it('renders correctly', () => {
    expect(toJson(renderComponent())).toMatchSnapshot();
  });
});
