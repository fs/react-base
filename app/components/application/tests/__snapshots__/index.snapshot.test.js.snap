// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application renders correctly 1`] = `
<div
  className="layout"
>
  <main
    className="wrapper"
  >
    <withRouter(Connect(Navigation)) />
    <ApplicationRoutes />
  </main>
  <Footer />
  <Connect(ModalContainer) />
  <SAlert
    html={true}
    stack={
      Object {
        "limit": 3,
      }
    }
  />
</div>
`;
